Web/Controller/Sessions.hs

```diff
@@ -9,4 +9,5 @@ instance Controller SessionsController where
     action CreateSessionAction = Sessions.createSessionAction @User
     action DeleteSessionAction = Sessions.deleteSessionAction @User
 
-instance Sessions.SessionsControllerConfig User where
\ No newline at end of file
+instance Sessions.SessionsControllerConfig User where
+    afterLoginRedirectPath = "/About"
```

Comment:
Is that action missing? :)

Reply:
Woops this may have been from some earlier hacking round with ihp-forum. Can just drop it.